src/widgets/school-menu/ui/school-menu/school-menu.tsx

```diff
@@ -12,13 +12,38 @@ type SchoolMenuProps = PropsWithChildren &
   HTMLProps<HTMLUListElement> & {
     heading?: string;
     color?: Color;
+    layout?: 'columns' | 'single';
+    isVisible?: boolean;
+    anchorLinks?: boolean;
   };
 
-export const SchoolMenu = ({ heading, color = 'light', children, className }: SchoolMenuProps) => {
+export const SchoolMenu = ({
+  heading,
+  color = 'light',
+  children,
+  className,
+  layout = 'single',
+  isVisible,
+  anchorLinks = false,
+}: SchoolMenuProps) => {
   return (
-    <div className={cx('school-menu')}>
+    <div
+      className={cx('school-menu', {
+        visible: isVisible === true,
+        hidden: isVisible === false,
```

Comment:
can we move all properties from .visible to .school-menu and add .hidden if isVisible===true?